src/components/Search/SearchComponent.tsx

```diff
@@ -101,12 +94,13 @@ function SearchComponent({
           </Tooltip>
         </InputLabel>
         {/**** Edit Button ****/}
-        {hasEditButton && !formIsDisplayed && (
+
+        {/* temporary until global edit button PR */}
+        {hasEditButton && !formIsDisplayed && isBaseComp && (
           <EditButton
             isBase={isBaseComp}
             onEditAction={() => {
               onEdit();
-              setFormIsDisplayed(true);
             }}
```

Comment:
optional:
```
onEditAction={onEdit}
```

Reply:
Yes, I left this here, but I'll just go ahead and add the Edit Entry global button in this PR too.